.github/workflows/build-release.yml

```diff
@@ -9,50 +9,68 @@ jobs:
   build-release:
     name: Build and push a tag image
     runs-on: ubuntu-18.04
-    env:
-      IMAGE_REGISTRY: quay.io/opdev
     steps:
     - uses: actions/checkout@v2
     - name: Set Env Tags
-      run: echo RELEASE_TAG=$(echo $GITHUB_REF | cut -d '/' -f 3) >> $GITHUB_ENV
+      run: |
+        echo RELEASE_TAG=$(echo $GITHUB_REF | cut -d '/' -f 3) >> $GITHUB_ENV
+        echo IMAGE_REGISTRY=$(echo ${{ secrets.IMAGE_REGISTRY }} | cut -d '/' -f 1) >> $GITHUB_ENV
+        echo IMAGE_REPO=$(echo ${{ secrets.IMAGE_REGISTRY }} | cut -d '/' -f 2) >> $GITHUB_ENV
 
     - name: Build Operator Image
       id: build-operator-image
       uses: redhat-actions/buildah-build@v2
       with:
-        image: ${{ env.IMAGE_REGISTRY }}/operator-certification-operator
+        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}/operator-certification-operator
         tags: ${{ env.RELEASE_TAG }}
         build-args: |
           release_tag=${{ env.RELEASE_TAG }}
         dockerfiles: |
           ./Dockerfile
 
-    - name: Build Bundle Image
-      id: build-bundle-image
-      uses: redhat-actions/buildah-build@v2
-      with:
-        image: ${{ env.IMAGE_REGISTRY }}/operator-certification-operator-bundle
-        tags: ${{ env.RELEASE_TAG }}
-        dockerfiles: |
-          ./bundle.Dockerfile
-
     - name: Push Operator Image
       id: push-operator-image
       uses: redhat-actions/push-to-registry@v2
       with:
         image: operator-certification-operator
         tags: ${{ steps.build-operator-image.outputs.tags }}
-        registry: ${{ env.IMAGE_REGISTRY }}
+        registry: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}
         username: ${{ secrets.REGISTRY_USER }}
         password: ${{ secrets.REGISTRY_PASSWORD }}
     
+    - name: Build Bundle
+      id: build-bundle
+      run: |
+        IMAGE_REGISTRY=${{ env.IMAGE_REGISTRY }} \
+        IMAGE_REPO=${{ env.IMAGE_REPO }} \
+        RELEASE_TAG=${{ env.RELEASE_TAG }} \
+        VERSION=${{ env.RELEASE_TAG }} \
+        USE_IMAGE_DIGESTS=true \
+        make bundle
+
+    # This is a very naive implementation. Mainly for PoC
+    - name: Add OpenShift metadata to bundle
+      id: add-openshift-metadata
+      run: |
+        echo '  com.redhat.openshift.versions: "v4.8-v4.10"' >> bundle/annotations.yaml
```

Comment:
This is a TAB right? So that the YAML is indented properly correct?

Reply:
Not a tab... DIdn't think it was a tab.

Reply:
They're spaces in the generated file.

20 20 is two spaces in the first line. And the last.

```
00000060  72 65 67 69 73 74 72 79  2b 76 31 0a 20 20 6f 70  |registry+v1.  op|
00000070  65 72 61 74 6f 72 73 2e  6f 70 65 72 61 74 6f 72  |erators.operator|
00000080  66 72 61 6d 65 77 6f 72  6b 2e 69 6f 2e 62 75 6e  |framework.io.bun|
00000090  64 6c 65 2e 6d 61 6e 69  66 65 73 74 73 2e 76 31  |dle.manifests.v1|
000000a0  3a 20 6d 61 6e 69 66 65  73 74 73 2f 0a 20 20 6f  |: manifests/.  o|
```